src/models/account.cpp

```diff
@@ -474,25 +477,29 @@ int Account::importFromCSV(const std::string& path)
             transaction.setAmount(amount);
             transaction.setGroupId(gid);
             addTransaction(transaction);
+
+            //Add Group if needed
+            if (getGroupById(gid) == std::nullopt && gid != -1) {
```

Comment:
Could you make if statements like this:
```cpp
if
{
}
```
instead of 
```cpp
if {
}
```
Just to keep styles consistent :)